includes/controller/CredentialsController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace BitCode\FI\controller;
+
+use BitCode\FI\Core\Util\HttpHelper;
+
+final class CredentialsController
+{
+    function getCredentials($params)  {
+        $apiEndpoint = 'https://auth-apps.bitapps.pro/apps/'. $params->actionName;
+        return $response = HttpHelper::get($apiEndpoint, null, null);
+        
+    }
+}
```

Comment:
- sanitize the actionName
- unnecessary $response variable
- remane this class to OneClickCredentialController